nncf/quantization/algorithms/weight_compression/openvino_backend.py

```diff
@@ -279,30 +303,6 @@ def _get_norm_weight_and_nf4_scale(
     return norm_weight, scale
 
 
-def _get_nf4_error(weight: np.ndarray, reduction_axes: Tuple[int], group_size: int = -1) -> float:
-    """
-    Calculates a quantity characterizing the difference between floating point weights and its nf4 fake quantized
-    (compressed and decompressed) version.
-
-    :param weight: Weight array to compress.
-    :param reduction_axes: Axis or axes along which to reduce (collect) different statistics (e.g. min, max).
-    :return: The quantity characterizing the nf4 error.
-    """
-    original_shape = weight.shape
-
-    norm_weight, scale = _get_norm_weight_and_nf4_scale(weight, reduction_axes, group_size)
-
-    index_of_quantile = np.searchsorted(CENTER_OF_NF4_QUANTILES, norm_weight)
-    nf4_rounded = NF4_QUANTILES[index_of_quantile]
```

Comment:
@andreyanufr removing NF4 error calculation, since we don't need it in 1/int8_err criteria.